index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Akronim&display=swap" rel="stylesheet">
+    <title>Fun with HTML5 Canvas</title>
+</head>
+
+<body>
+    <div class="page-title">
+        <h1>Fun with HTML5 Canvas</h1>
+    </div>
+
+    <div class="button">
+        <h2>clear</h2>
```

Comment:
Do not forget about sematic html. 
Each tag had its own meaning. 
H1-H6 tags are for headers, for buttons there is special tag button

Reply:
I totally agree. Change all divs on semantic tags.